README.md

```diff
@@ -16,11 +16,10 @@
 
 # 📣 What does this dbt package do?
 
-This package models AWS Cloud Cost data from [Fivetran's connector](https://fivetran.com/docs/applications/aws_cloud_cost). It uses data in the format described by [this ERD](https://fivetran.com/docs/applications/aws_cloud_cost#schemainformation).
+This package models AWS Cloud Cost data from [Fivetran's connector](https://fivetran.com/docs/connectors/applications/aws-cost-report). 
 
 The main focus of the package is to transform the core object tables into analytics-ready models, including:
-<!--section="aws_cloud_cost_model"-->
-  - Materializes [AWS Cloud Cost staging tables](https://fivetran.github.io/dbt_aws_cloud_cost/#!/overview/aws_cloud_cost_source/models/?g_v=1) which leverage data in the format described by [this ERD](https://fivetran.com/docs/applications/aws-cost-report/#schemainformation). These staging tables clean, test, and prepare your AWS Cloud Cost data from [Fivetran's connector](https://fivetran.com/docs/applications/aws-cost-report) for analysis by doing the following:
+  - Materializes [AWS Cloud Cost staging tables](https://fivetran.github.io/dbt_aws_cloud_cost/#!/overview/aws_cloud_cost_source/models/?g_v=1) which leverage data in the format described by . These staging tables clean, test, and prepare your AWS Cloud Cost data from [Fivetran's connector](https://fivetran.com/docs/connectors/applications/aws-cost-report) for analysis by doing the following:
```

Comment:
Same note about needing to remove the ERD reference and also the link needed to be updated for the connector docs.